VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -111,8 +107,23 @@ public void onReceivedError(WebView view, int errorCode,
                 hideLoadingProgress();
                 pageIsLoaded = false;
                 ViewPagerDownloadManager.instance.done(CourseUnitWebviewFragment.this, false);
-                super.onReceivedError(view, errorCode, description, failingUrl);
             }
+
+            // Restore these annotations when we upgrade our compile SDK version to Marshmallow
```

Comment:
Hmm, so the inspection is by default a warning. I could turn it into an error instead, but we have 138 violations of the rule in the code base :frowning: so that could become an issue. Also currently we have `abortOnError false` set in the `lintOptions` in the `build.gradle` file, so there might be lots of unaddressed errors as well. I think we can just enable it as a warning instead for the moment, and rely on the TODO comment and the warning for the annotations to be applied in some cleanup process.